src/arch/microblaze/kernel/irq_handler.c

```diff
@@ -25,20 +25,20 @@ void irq_handler(void) {
 		unsigned int irq_num;
 		for (irq_num = 0; irq_num < IRQCTRL_IRQS_TOTAL; irq_num++) {
 			if (irqctrl_pending(irq_num)) {
-				/* disable interrupt and clear it later, since ack
-					 * have no effect on level interrupt */
+				/* disable interrupt and clear it later, since ACK
+				 * have no effect on level interrupt */
 				irqctrl_disable(irq_num);
 
-				/*now we allow nested irq*/
-				ipl_enable();
+				/*now we allow nested IRQ*/
+				//ipl_enable();
```

Comment:
I'm angry. I create an issue before this PR will merged if we don't resolve this problem